Atlas/android/app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,9 @@
 <resources>
     <string name="app_name">Atlas</string>
+    <string name="enhance_the_culture">Enhance the Culture</string>
+    <string name="together">Together</string>
+    <string name="learn_more_button">Learn more</string>
+    <string name="login_button">Login</string>
+    <string name="or">or</string>
+    <string name="register_button">Register</string>
```

Comment:
To be consistent with mockups you should use Signup instead of register.  Also you can change the "register_button" as "signup_button".

Reply:
We have register in the android mockups. Why should we use signup?

Reply:
My bad, I accidentally looked at the web mockups.